### ISSUE TITLE
checkpatch warnings in for-later branch

### ISSUE THREAD

**Issue by mriswyth** [davidker/unisys#51]
When running format-patch on the for-later branch the following totals showed up:

total: 0 errors, 0 warnings, 0 checks, 88 lines checked
**total: 0 errors, 7 warnings, 0 checks, 106 lines checked**
total: 0 errors, 0 warnings, 0 checks, 42 lines checked
**total: 0 errors, 3 warnings, 0 checks, 113 lines checked**
total: 0 errors, 0 warnings, 0 checks, 25 lines checked
total: 0 errors, 0 warnings, 0 checks, 41 lines checked
total: 0 errors, 0 warnings, 0 checks, 10 lines checked
total: 0 errors, 0 warnings, 0 checks, 60 lines checked
total: 0 errors, 0 warnings, 0 checks, 181 lines checked
total: 0 errors, 0 warnings, 0 checks, 145 lines checked
total: 0 errors, 0 warnings, 0 checks, 70 lines checked
total: 0 errors, 0 warnings, 0 checks, 8 lines checked
total: 0 errors, 0 warnings, 0 checks, 62 lines checked
total: 0 errors, 0 warnings, 0 checks, 61 lines checked
total: 0 errors, 0 warnings, 0 checks, 82 lines checked
total: 0 errors, 0 warnings, 0 checks, 241 lines checked
total: 0 errors, 0 warnings, 0 checks, 7 lines checked
total: 0 errors, 0 warnings, 0 checks, 32 lines checked
total: 0 errors, 0 warnings, 0 checks, 9 lines checked

The 7 warnings come from patch: 37221bdd361404ba5586babd62b548fdcb798702
[staging: unisys: visorinput: remove need for 'depends on FB'](37221bdd361404ba5586babd62b548fdcb798702)

The 3 warnings come from patch: f7adbfb8afee6855bc221b5a8ce80080cff77b9b
[staging: unisys: visorinput: respond to resolution changes on-the-fly](f7adbfb8afee6855bc221b5a8ce80080cff77b9b)

**Comment by mriswyth** [comment]
Comment from @davidker:

> Ideally the output of this is 2 patches that I can squash to the other patches. Please DO NOT put all the fixes in one patch. That makes my job as hard as just fixing the patches

**Comment by mriswyth** [comment]
@davidker 
The only way that I could come up with to make this so you didn't have to do the work was to rebase for-later and fix the two patches. I have pushed this as the githubissue#51 branch. Of course this rewrites the hashes of the first patch I fixed and all subsequent so my thinking was that this would require replacing for-later with githubissue#51.

Verified that the patch differences started at the first patch with warnings and the subsequent patches were only different by changing unsigned to unsigned int, fixing lines due to longer text, hashes, and line numbers.

```
mkdir ../patches
mkdir ../patches/for-later
mkdir ../patches/githubissue#51
git checkout -b for-later origin/for-later
git format-patch -o ../patches/for-later 9136d04fe9255522c4c8ef5124be899906de7dd1
git checkout -b githubissue#51 origin/githubissue#51
git format-patch -o ../patches/githubissue#51 9136d04fe9255522c4c8ef5124be899906de7dd1
cd ../patches
diff -Naur for-later/ githubissue#51 > issue51.diff
```

Verified patches show no checkpatch notices.
`for p in ../patches/for-later/*.patch; do scripts/checkpatch.pl --strict --no-tree --terse "$p"; done;`
Shows the original 10 warnings.
`for p in ../patches/githubissue#51/*.patch; do scripts/checkpatch.pl --strict --no-tree --terse "$p"; done;`
Shows nothing.
